python/tests/index/merger_binary_test.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Usage: py.test tests
+
+import keyvi._pycore
+
+import os.path
+import subprocess
+import sys
+
+
+def get_package_root():
+    module_location = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(keyvi._pycore.__file__)), ".."))
+
+    if (sys.version_info.major >= 3):
+        module_location = module_location.encode('utf-8')
+
+    return module_location
+
+
+def get_interpreter_executable():
```

Comment:
Maybe I'm missing something, but as in `Index.pyx` we are using `get_interpreter_executable()` and `get_package_root()` from `keyvi._pycore`. It seems it's better to use them here (and have tested) as well. 

Maybe even have a function returning `get_interpreter_executable() + b" " + os.path.join(get_package_root(), b"_pycore" , b"keyvimerger.py")`, so we are sure that tests and merger are utilizing same code.

@hendrikmuhs what do you think about ?

Reply:
I was thinking that both are internal functions, not available from external.

But I managed to find them, they are in keyvi._core, so yes,makes sense, I will change it.